assets/src/scripts/store/index.js

```diff
@@ -479,6 +479,17 @@ export const Actions = {
             type: 'SET_DAILY_VALUE_CURSOR_OFFSET',
             cursorOffset
         };
+    },
```

Comment:
Could you add tests for these? Then it can be merged.

Reply:
I added a couple tests, hopefully good now.